Tribler/Test/Core/CreditMining/test_creditmining_sys.py

```diff
@@ -285,17 +274,17 @@ def setUpPreSession(self):
         # we use dummy dispersy here
         self.config.set_dispersy(False)
 
-    @blocking_call_on_reactor_thread
-    def create_fake_allchn_comm(self):
-        """
-        This method creates a fake AllChannel community so we can check whether a request is made in the community
-        when doing stuff with a channel.
-        """
-        self.session.lm.dispersy._database.open()
-        fake_member = DummyMember(self.session.lm.dispersy, 1, "a" * 20)
-        member = self.session.lm.dispersy.get_new_member(u"curve25519")
-        fake_community = AllChannelCommunity.init_community(self.session.lm.dispersy, fake_member, member)
-        self.session.lm.dispersy._communities = {"allchannel": fake_community}
+    # @blocking_call_on_reactor_thread
```

Comment:
This is outdated diff. I removed this and use @devos50 part of code instead (and refer it).